src/solve.jl

```diff
@@ -81,7 +81,7 @@ function DiffEqBase.__init(
 
   t = tspan[1]
 
-  if (((!(typeof(alg) <: OrdinaryDiffEqAdaptiveAlgorithm) && !(typeof(alg) <: OrdinaryDiffEqCompositeAlgorithm)) || !adaptive) && dt == tType(0) && isempty(tstops)) && !(typeof(alg) <: FunctionMap)
+  if (((!(typeof(alg) <: OrdinaryDiffEqAdaptiveAlgorithm) && !(typeof(alg) <: OrdinaryDiffEqCompositeAlgorithm)) || !adaptive) && dt == tType(0) && isempty(tstops)) && !(typeof(alg) <: Union{FunctionMap,LinearExponential})
```

Comment:
Why is this needed? `isempty(tstops)` should catch it?

Reply:
ooooh, I see. You want it to accept no `dt` so that it just solves with `dt` equal to the timespan? That makes sense.